src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.spec.ts

```diff
@@ -90,17 +113,18 @@ describe('DotEditPageWorkflowsActionsComponent', () => {
                 firstButton = splitButtons[0].nativeElement;
                 secondButton = splitButtons[1].nativeElement;
                 thirdButton = splitButtons[2].nativeElement;
+                fixture.detectChanges();
             });
 
             it('should fire actions on click in the menu items', () => {
                 firstButton.click();
-                expect(dotWorkflowService.fireWorkflowAction).toHaveBeenCalledWith(component.inode, mockWorkflows[0].id);
+                expect(dotWorkflowService.fireWorkflowAction).toHaveBeenCalledWith(component.page.workingInode, mockWorkflows[0].id);
```

Comment:
If you are keeping the property inode in the class... why did you check the ```page.workingInode``` here?